src/components/scenes/FioRequestListScene.js

```diff
@@ -173,16 +173,27 @@ class FioRequestList extends React.Component<Props, LocalState> {
     return nextFioRequests
   }
 
-  showNoBundlesAlert = () =>
-    Airship.show(bridge => (
+  showNoBundlesAlert = async (fioWallet: EdgeCurrencyWallet, fioAddressName: string) => {
```

Comment:
Please change all variables that have `bundles` to `bundledTxs` or `bundled_txs` bundles is not a proper name